app/representers/cms/page_representer.rb

```diff
@@ -1,32 +1,28 @@
+# frozen_string_literal: true
+
 module CMS::PageRepresenter
   include ThreeScale::JSONRepresenter
 
-  wraps_resource
+  wraps_resource ->(*) { self.class.data_tag }
 
   property :id
-
-  with_options(if: ->(options) { options[:short] == false }) do |p|
-    p.property :draft, render_nil: true
-    p.property :published, render_nil: true
-  end
-
+  property :created_at
+  property :updated_at
+  property :title
   property :system_name
-  property :liquid_enabled
+  property :layout_id
```

Comment:
Doesn't appear when empty. The field should be returned, but empty. In JSON the way to return emptiness is `""`, right?